.jshintrc

```diff
@@ -14,7 +14,7 @@
   "quotmark": "single",
   "undef": true,
   "unused": true,
-  "strict": true,
+  "strict": false,
```

Comment:
We are still using strict mode - it appears at the top of the dist file. I removed it from the jshint rules because it was insisting we put it at the top of every file in src.